tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/NLogTests.cs

```diff
@@ -61,7 +61,7 @@ public void DoesNotInjectLogsWhenDisabled(string packageVersion)
             var expectedCorrelatedSpanCount = 0;
 
             using (var agent = EnvironmentHelper.GetMockAgent())
-            using (RunSampleAndWaitForExit(agent.Port, packageVersion: packageVersion))
+            using (RunSampleAndWaitForExit(agent, packageVersion: packageVersion))
```

Comment:
😌 The relief when you realise most of the 116 file changes are `agent.Port` -> `agent`